sectionfilter.c

```diff
@@ -270,54 +271,68 @@ cSatipSectionFilterHandler::~cSatipSectionFilterHandler()
       Delete(i);
 }
 
+bool cSatipSectionFilterHandler::Send(void)
+{
+  // zero polling structures
+  memset(pollFdsM, 0, sizeof(pollFdsM));
+
+  // assemble all handlers to poll
+  for (unsigned int i = 0; i < eMaxSecFilterCount; ++i) {
+      if (filtersM[i] && filtersM[i]->Available() != 0) {
+         pollFdsM[i].fd = filtersM[i]->GetFd();
+         pollFdsM[i].events = POLLOUT;
+         }
+      }
+
+  // anyone ready for writing
+  if (poll(pollFdsM, eMaxSecFilterCount, 10) <= 0)
+     return false;
+
+  // send data
+  for (unsigned int i = 0; i < eMaxSecFilterCount; ++i) {
+      if(pollFdsM[i].revents & POLLOUT)
```

Comment:
Missing whitespace after 'if'

Reply:
It's not my natural style of coding. But I'm getting better, ...

Reply:
I'm trying to keep the way of KLS here (except for parameter/member naming).